src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/events/CiagInductionTimelineEventTest.kt

```diff
@@ -0,0 +1,72 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.service.events
+
+import org.awaitility.kotlin.await
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.verify
+import software.amazon.awssdk.services.sns.model.MessageAttributeValue
+import software.amazon.awssdk.services.sns.model.PublishRequest
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.aValidPrisonNumber
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.TimelineEventType
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.assertThat
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.service.events.aValidCiagInductionCreatedEvent
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.service.events.aValidCiagInductionUpdatedEvent
+
+internal class CiagInductionTimelineEventTest : IntegrationTestBase() {
+
+  @Test
+  fun `should process CIAG induction created event`() {
+    // Given
+    val prisonNumber = aValidPrisonNumber()
+    val inductionCreatedEvent = aValidCiagInductionCreatedEvent(prisonNumber = prisonNumber)
+
+    // When
+    inboundSnsClient.publish(
```

Comment:
I'm finding the terminology difficult here (I know its not your naming)

"inbound" to me suggests something coming in to our service ... but you are publishing on it, which suggests its "outbound" ....

I wonder if we can come up with better terms?

Reply:
Perhaps just `snsClient` ?

Reply:
mmm...I'm not sure. It's in inbound topic into our system. Yes we're publishing a message to it here, but that's solely to test it. If we ever have an outbound topic then `snsClient` wouldn't work.

Reply:
Have a look at the example app's config here: https://github.com/ministryofjustice/hmpps-spring-boot-sqs/blob/main/test-app/helm_deploy/hmpps-template-kotlin/values.yaml#L54C1-L54C1